stl/inc/chrono

```diff
@@ -2209,6 +2221,2949 @@ namespace chrono {
 
         return hours{_Ret};
     }
+
+    // [time.zone.info]
+
+    // STRUCT sys_info
+    struct sys_info {
+        sys_seconds begin;
+        sys_seconds end;
+        seconds offset;
+        minutes save;
+        string abbrev;
+    };
+
+    // STRUCT local_info
+    struct local_info {
+        static constexpr int unique      = 0;
+        static constexpr int nonexistent = 1;
+        static constexpr int ambiguous   = 2;
+
+        int result;
+        sys_info first;
+        sys_info second;
+    };
+
+    // CLASS nonexistent_local_time
+    class nonexistent_local_time : public runtime_error {
+    public:
+        template <class _Duration>
+        nonexistent_local_time(const local_time<_Duration>&, const local_info&)
+            : runtime_error("TRANSITION: work in progress") {}

```

Comment:
WG21-N4885 [time.zone.exception.nonexist]/3 specifies this in terms of `operator<<` which @mnatsuhara is working on, so this will be completed very soon. There's a card in the [Chrono Project](https://github.com/microsoft/STL/projects/7?fullscreen=true) tracking this.